### DISCUSSION TITLE
space-x/y util and type="hidden"

### DISCUSSION THREAD

**Discussion by madsenmm** [tailwindlabs/tailwindcss#9553] [Ideas]
Would it be possible to exclude `<input type="hidden">`, from being affected by `space-x-` and `space-y-` utils?

So instead of:
```CSS
.selector :not([hidden]) ~ :not([hidden])
```

It becomes
```CSS
.selector :not([type='hidden']):not([hidden]) ~ :not([hidden])
```

**Comment by adamwathan** [comment]
Going to stick with how it is — adding any other bits to the selector increases the specificity and it's just a real rabbit hole trying to make this totally bullet proof. Best bet is just add the `hidden` attribute to those inputs, or use `gap` which is better for all of this stuff and has good browser support these days.
